docs/content_management/pages/react_app_block.md

```diff
@@ -82,7 +82,10 @@ In the following example, you learn how to create `Calculator` React App block.
 
 ### Configure React App Block
 
-First, create a .jsx file which describes your component.
+First, install the react.
+Run `yarn add react` command.
```

Comment:
I don't have to do that on a 4.5.3 to make the example works.
By executing `yarn list --pattern react`, I see that it comes with `react@18.2.0`.
But it doesn't harm to ensure it's there.